girder_annotation/girder_large_image_annotation/web_client/views/annotationListWidget.js

```diff
@@ -182,24 +193,95 @@ const AnnotationListWidget = View.extend({
+            .prop('disabled', !anySelected)
+            .toggleClass('disabled', !anySelected)
+            .css('color', anySelected ? '' : 'grey');
+        if (!id) {
+            if ($el.find('input').is(':checked')) {
+                allChecks.prop('checked', true);
+            }
+            if (!$el.find('input').is(':checked')) {
+                allChecks.prop('checked', false);
+            }
+        }
+    },
+
+    _downloadSelectedAnnotations(evt) {
+        evt.preventDefault();
+
+        const selectedAnnotations = this.$('.g-annotation-select input:checked')
+            .closest('.g-annotation-row')
+            .map((_, el) => $(el).data('annotationId'))
+            .get();
+
+        selectedAnnotations.forEach((id) => {
+            const annotation = this.collection.get(id);
+            if (annotation) {
+                annotation.fetch().then(() => {
+                    const url = `${getApiRoot()}/annotation/${annotation.id}`;
+                    const downloadAnchor = document.createElement('a');
+                    downloadAnchor.setAttribute('href', url);
+                    downloadAnchor.setAttribute('download', `${annotation.get('annotation').name}.json`);
+                    document.body.appendChild(downloadAnchor);
+                    downloadAnchor.click();
+                    document.body.removeChild(downloadAnchor);
+                    return null;
+                });
+            }
+        });
+    },
+
     _deleteAnnotation(evt) {
         const $el = $(evt.currentTarget);
         const id = $el.parents('.g-annotation-row').data('annotationId');
+        const checkedAnnotations = this.$el.find('.g-annotation-select input[type=checkbox]:checked');
+        const checkedAnnotationIds = [];
         if (!id) {
-            confirm({
-                text: `Are you sure you want to delete <b>ALL</b> annotations?`,
-                escapedHtml: true,
-                yesText: 'Delete',
-                confirmCallback: () => {
-                    restRequest({
-                        url: `annotation/item/${this.model.id}`,
-                        method: 'DELETE'
-                    }).done(() => {
-                        this.collection.fetch(null, true);
-                    });
-                }
-            });
-            return;
+            for (let i = 0; i < checkedAnnotations.length; i++) {
+                const annotationId = $(checkedAnnotations[i]).parents('.g-annotation-row').data('annotationId');
+                checkedAnnotationIds.push(annotationId);
+            }
+            if (checkedAnnotations.length !== 0) {
+                confirm({
+                    text: `<h3>Are you sure you want to delete the following annotations?</h3>
+                        <ul
+                          style="max-height: 200px; padding-left: 0; overflow-y: auto;"
```

Comment:
It is better to style via css classes than inline, because we can reuse components or have them appear in different places.  Also, I think this is better without the max-height restriction, since the dialog's max-height will be restricted elsewhere.  Otherwise, use a form like `calc(100vh - 200px)` so it can fill more of the screen (when I have a lot of annotations, it seems sub-optimal to scroll when it could all be visible).